Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.5
 
+RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
```

Comment:
Is it good idea to conditionally add/edit `hosts: files dns` than forceful append?